phone/src/apps/match/components/views/MatchList.tsx

```diff
@@ -21,7 +22,8 @@ function MatchList() {
   const Nui = useNuiRequest();
   const classes = useStyles();
   const { t } = useTranslation();
-  const { matches, error } = useMatches();
+  const matches = useMatchesValue();
+  const error = useRecoilValue(matchState.errorLoadingMatches);
```

Comment:
Might be better to just wrap the matchmaker app in an error boundary component and control error state handling that way

Reply:
Abstract with a hook within the state file to keep consistent with conventions